src/main/scala/com/codacy/client/bitbucket/v2/service/CommitServices.scala

```diff
@@ -25,20 +27,28 @@ class CommitServices(client: BitbucketClient) {
 
     val values = JsObject(params.toSeq :+ "content" -> Json.obj("raw" -> JsString(body)))
 
-    client.postJson(Request(url, classOf[CommitComment]), values)
+    client.postJson(Request(commitCommentUrl, classOf[CommitComment]), values)
   }
 
-  def listComments(author: String, repo: String, commit: String): RequestResponse[Seq[CommitComment]] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$author/$repo/commit/$commit/comments"
+  def listComments(author: String, repository: String, commit: String): RequestResponse[Seq[CommitComment]] = {
+    val commitCommentUrl = generateCommitCommentUrl(author, repository, commit)
 
     client
-      .executePaginated(Request(url, classOf[Seq[CommitComment]]))
+      .executePaginated(Request(commitCommentUrl, classOf[Seq[CommitComment]]))
       .map(_.filterNot(_.deleted))
   }
 
-  def deleteComment(author: String, repo: String, commit: String, commentId: Long): RequestResponse[Boolean] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$author/$repo/commit/$commit/comments/$commentId"
+  def deleteComment(author: String, repository: String, commit: String, commentId: Long): RequestResponse[Boolean] = {
+    val commitCommentUrl = generateCommitCommentUrl(author, repository, commit)
+    val url = s"$commitCommentUrl/$commentId"
 
     client.delete(url)
   }
+
+  private def generateCommitCommentUrl(author: String, repository: String, commit: String): String = {
+    val encodedAuthor = URLEncoder.encode(author, "UTF-8")
```

Comment:
Codacy found an issue: [The string literal 'UTF-8' appears 3 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)